<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtry_Nieliniowe</class>
 <widget class="QWidget" name="Filtry_Nieliniowe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pMediana">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Mediana</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pMinimum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pMaximum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pAnuluj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>106</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Cofnij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pOk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyj≈õcie</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>43</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Maska</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
